client/app/i18n/locales/de/vizlib.json

```
@@ -0,0 +1,261 @@
+{
+  "Cancel": "Abbrechen",
+  "Apply": "Anwenden",
+  "Something went wrong": "Etwas ist schief gelaufen",
+  "Align left": "Linksbündig",
+  "Align center": "Zentriert",
+  "Align right": "Rechtsbündig",
```

here is a suggestion
```suggestion
  "Align right": "test",
```